src/main/java/org/sobotics/guttenberg/commands/CheckInternet.java

```diff
@@ -31,11 +31,11 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 
-import fr.tunaki.stackoverflow.chat.Message;
-import fr.tunaki.stackoverflow.chat.Room;
+import org.sobotics.chatexchange.chat.Message;
+import org.sobotics.chatexchange.chat.Room;
 
 /**
- * Command to search for plagarisim using internet
+ * Command to search for plagiarisim using internet
```

Comment:
The spelling is still wrong.

Reply:
Should have plagiarized it from the description on the plagiarism detector. https://github.com/SOBotics/Guttenberg.